api/envoy/admin/v3/server_info.proto

```diff
@@ -85,8 +85,6 @@ message CommandLineOptions {
 
   reserved 12, 20, 21;
 
-  reserved "max_stats", "max_obj_name_len";
```

Comment:
Why are they no longer reserved?

Reply:
maybe not for a great reason, but if I back out this diff it will break OptionsImplTest, which accounts for reserved names. I'm fine either way.

What's the benefit of reserving them; to ensure we don't re-add the same named option later?

Reply:
Yeah, that's the idea; largely because this is a huge issue with JSON/YAML output formats. If it's just for a single test, I'd argue just fixing the test, although I don't see a high chance of reusing these names.

Reply:
So basically the state of the PR as of https://github.com/envoyproxy/envoy/pull/11740/commits/debf454397f19903197f2d1f7d334f0b53cd6536
is what you'd vote for?

Reply:
Sure; if you wanted to get fancy you could probably figure out what is reserved via reflection, but a simple `-2` is fine for me.